js/app/packages/app/index.css

```
@@ -183,6 +183,7 @@
   --animate-slide-in: slideIn 150ms cubic-bezier(0.16, 1, 0.3, 1);
   --animate-typing-dot: typing-dot 1s infinite;
   --animate-swipe-out: swipeOut 100ms ease-out;
+  --animate-indeterminate-bar: indeterminateBar 1.1s ease-in-out infinite;
```

Stylelint (17.12.0)

[error] Expected "indeterminateBar" to be "indeterminatebar" (value-keyword-case)

(value-keyword-case)